src/services/meetings/meetings-util.ts

```diff
@@ -0,0 +1,86 @@
+import { SupabaseDB } from "../../supabase"; // Supabase database
+import {
+    meetingView,
+    createMeetingValidator,
+    updateMeetingValidator,
+} from "./meetings-schema";
+
+export const getAllMeetings = async () => {
+    console.log("Getting all meetings from Supabase");
+    const { data, error } = await SupabaseDB.MEETINGS.select("*");
```

Comment:
if you're only returning 3 things, just fetch those 3 from database, and that makes ur life a lot easier